src/components/CloseLink.js

```diff
@@ -24,14 +24,14 @@ type Props = {
 
 class CloseLink extends React.Component<Props> {
   onClick = event => {
+    event.preventDefault();
+    event.stopPropagation();
```

Comment:
Should this be handled by the component in every case? Maybe a onClick prop is given to overwrite exactly this. No?

Reply:
Yeah, maybe you are right @lennerd. I didn't write it just moved it up, so the logic works.